concordia/views.py

```diff
@@ -67,6 +55,8 @@ def healthz(request):
     # at last once:
     status["database_has_data"] = Campaign.objects.count() > 0
 
+    status["application_version"] = get_version()
```

Comment:
I'm wondering whether there's enough of a performance impact here that we should toss `get_version()` into a function decorated with `functools.lru_cache` – is there a chance that this fails back to querying Git or does it just ask pkg resources?